tests/Testcontainers.Tests/Unit/Configurations/CustomConfigurationTest.cs

```diff
@@ -32,6 +35,49 @@ public void GetDockerHostCustomConfiguration(string propertyName, string propert
         Assert.Equal(expected, customConfiguration.GetDockerHost()?.ToString());
       }
 
+      [Theory]
+      [InlineData("", "", null)]
+      [InlineData("DOCKER_TLS", "", null)]
+      [InlineData("DOCKER_TLS", "0", false)]
+      [InlineData("DOCKER_TLS", "false", false)]
+      [InlineData("DOCKER_TLS", "FALSE", false)]
+      [InlineData("DOCKER_TLS", "1", true)]
+      [InlineData("DOCKER_TLS", "TRUE", true)]
+      [InlineData("DOCKER_TLS", "true", true)]
+      public void GetDockerTlsCustomConfiguration(string propertyName, string propertyValue, bool? expected)
+      {
+        SetEnvironmentVariable(propertyName, propertyValue);
```

Comment:
This is something what I want to discuss with you. Becasue in case that something is set up via ENV variables then this  and other tests can break. Which approach is est to use?

Reply:
This shouldn't be a problem. We can even improve this, by adding the target to `SetEnvironmentVariable`: `Environment.SetEnvironmentVariable(propertyName, propertyValue, EnvironmentVariableTarget.Process)`. The tests are executed sequentially at the end (no other test will run).